todolist/date_filter.go

```diff
@@ -144,7 +142,7 @@ func (f *DateFilter) filterOverdue(pivot time.Time) []*Todo {
 }
 
 func (f *DateFilter) FindSunday(pivot time.Time) time.Time {
-	switch now.New(pivot).Weekday() {
+	switch time.Now().Weekday() {
```

Comment:
the only issue here is that previously `now.New` was using the `pivot` time to do calculations.  Your version  uses `time.now()` which can make the tests more brittle.  Can you change it to use `pivot`?